component-monitor/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "component-monitor"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+clap = { version = "4", default-features = false, features = ["derive", "env", "help", "std", "usage"] }
```

Comment:
Really it's just for keeping it tight. It's not really a CLI application, just an internal service.

I dropped `default_features` across the board b/c I dislike things creeping up on me, is all ;)

And Innes warned me about `nix` being heavy, so that's probably why ;)

Reply:
Ah, this sounds really good, thanks for the clarification!

Reply:
Ah that sounds good, thanks for the clarification!